flink-connectors/flink-connector-aws-kinesis-data-streams/src/main/java/org/apache/flink/streaming/connectors/kinesis/util/AwsV2Util.java

```diff
@@ -110,6 +114,20 @@ public static SdkAsyncHttpClient createHttpClient(
                         .map(Duration::ofMillis)
                         .orElse(DEFAULT_EFO_HTTP_CLIENT_READ_TIMEOUT);
 
+        boolean trustAllCerts =
+                Optional.ofNullable(
+                                consumerConfig.getProperty(
+                                        AWSConfigConstants.TRUST_ALL_CERTIFICATES))
+                        .map(Boolean::parseBoolean)
+                        .orElse(DEFAULT_TRUST_ALL_CERTIFICATES);
+
+        Protocol httpProtocolVersion =
+                Optional.ofNullable(
+                                consumerConfig.getProperty(
+                                        AWSConfigConstants.HTTP_PROTOCOL_VERSION))
+                        .map(Protocol::valueOf)
+                        .orElse(DEFAULT_HTTP_PROTOCOL);
```

Comment:
Why is this change needed? To me, it is not obvious why the sink needs it. Can you put this on a separate commit?

Reply:
These are needed for Kinesalite to run.